programs/alexandria/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-add_library( alexandria MODULE alexandria.cpp alexandria.hpp )
+add_library( alexandria SHARED alexandria.cpp alexandria.hpp )
```

Comment:
nie je module lepsie?  MODULE libraries are plugins that are not linked into other targets but may be loaded dynamically at runtime using dlopen-like functionality.